pkg/api/v20240812preview/openshiftcluster_validatestatic.go

```diff
@@ -455,6 +479,9 @@ func (sv openShiftClusterStaticValidator) validatePlatformWorkloadIdentityProfil
 		return nil
 	}
 
+	// collect operator names to check for duplicates
+	operatorNames := []string{}
```

Comment:
Can we handle the duplication check using a map?

Reply:
I suppose so? Would that accomplish something different?

Reply:
No, nothing from functionality pov, but I believe from complexity pov.